docs/en/latest/architecture-design/global-rule.md

```diff
@@ -21,7 +21,7 @@ title: Global rule
 #
 -->
 
-[Plugin](plugin.md) just can be bound to [Service](service.md) or [Route](route.md), if we want a [Plugin](plugin.md) work on all requests, how to do it?
+The [Plugin](plugin.md) configuration can be bound directly to [Route](route.md) or it can be bound to [Service](service.md) or [Consumer](consumer.md), if we want a [Plugin](plugin.md) work on all requests, how to do it?
```

Comment:
```suggestion
The [Plugin](plugin.md) configuration can be bound directly to a [Route](route.md), or to a [Service](service.md) or [Consumer](consumer.md), What if we want a [Plugin](plugin.md) to work on all requests, how to do it?
```

Reply:
Yes, I think it's better. But it raise a new question, I referred [plugin.md](https://github.com/apache/apisix/blob/master/docs/en/latest/architecture-design/plugin.md) to translate the file, so it is possible to update the [plugin.md](https://github.com/apache/apisix/blob/master/docs/en/latest/architecture-design/plugin.md) together ?

Reply:
I afraid the answer is no. **One PR for one thing**, PR's destination is to update `global_rule`, how about do it in another PR ?

Reply:
OK